benchmark-utils_test.go

```diff
@@ -75,6 +137,8 @@ func runGetObjectBenchmark(b *testing.B, obj ObjectLayer, objSize int) {
 	b.StopTimer()
 
 }
+
+// picks a random byte and repeats it to size bytes.
 func generateBytesData(size int) []byte {
 	// randomly picks a character and returns its equivalent byte array.
 	getRandomByte := func() []byte {
```

Comment:
isn't getRandomByte() useful outside of this function? - why is it kept like lambda?